packages/interactivity-router/src/assets/dynamic-importmap/loader.ts

```diff
@@ -0,0 +1,337 @@
+/**
+ * External dependencies
+ */
+import * as lexer from 'es-module-lexer';
+
+/**
+ * Internal dependencies
+ */
+import { fetchModule } from './fetch';
+import { resolve } from './resolver';
+
+export interface ModuleLoad {
+	u?: string; // original URL
+	r?: string; // response url
+	f?: Promise< ModuleLoad >; // fetch promise
+	S?: string; // source code
+	L?: Promise< void >; // link-promise (dependency fetch)
+	a?: ReturnType< typeof lexer.parse >; // analysis ([ imports, exports, ... ])
+	d?: ModuleLoad[]; // deps
+	b?: string; // blobUrl
+	s?: string; // shellUrl for circular references
+}
```

Comment:
Do we need to keep this one letter property names?

Reply:
I've changed the names in https://github.com/WordPress/gutenberg/pull/70353/commits/c1fb03e5c465f7be4b1a70bc2e011168daac85c6.

The bundle size changes by a hundred bytes, but it's barely noticeable when compressed with gzip.

These are the size differences in `interactivity-router/index.min.js`:
| Commit | Without compression | With gzip compression |
| --- | --- | --- |
| c1fb03e5c465f7be4b1a70bc2e011168daac85c6 (after) | 24.3 KiB (24837 B) | 11.5 kB (11110 B) |
| 9e1606e2ee4a6d714d6073ef8080c061b632055d (before) | 24 KiB (24549 B) | 11.4 kB (11063 B) |

Reply:
Thanks. I would leave it like that in the codebase and leave that optimization to the JS minifier.